### ISSUE TITLE
Quick connect support

### ISSUE THREAD

**Issue by viggy96** [jarnedemeulemeester/findroid#696]
It would be nice to be able to use Findroid to login to other devices via quick connect. Possibly even use quick connect to login to Findroid itself

**Comment by Natanel-Shitrit** [comment]
What do you mean by "_login to other devices via quick connect_"?
Findroid already supports Jellyfin quick connect.

**Comment by viggy96** [comment]
Sorry about that, I realised that it already does after I posted this lol. The first part would be nice though, like having a place to enter a quick connect code from another device, and entering it into Findroid to authenticate.

**Comment by Natanel-Shitrit** [comment]
It's a nice feature, but I don't think it should be added to Findroid.

Findroid does not aim to have the administrative tools like the official client.

Although it's one of the things that I might consider adding since it's less "administrative" than other things that were rejected for the same exact reason.

**Comment by viggy96** [comment]
Yeah I agree, Findroid shouldn't have the full set of administrative tools. 
But I see this as just another aspect of enabling login.

**Comment by viggy96** [comment]
Has any thought been given to having a place to enter quick connect codes from other devices?

**Comment by viggy96** [comment]
Any thought to having this feature? 

As it stands, even if a user doesn't want to do administrative Jellyfin tasks on a phone, you still need the official Jellyfin app to entering quick connect codes from other devices, since Findroid doesn't support that. 

Having this feature would allow users who just need to watch content to only have the Findroid app.

**Comment by Rafficer** [comment]
I'd also highly appreciate that option. I only have findroid on my tablet and phone as it works way better than the jellyfin app, and when I authenticate on a different device (like at a friends place), I don't have to whip out a browser or get the official app.

I don't think it's an "administrative tool" per se. It's just a convenience feature that would be great for a mobile client.
